src/Tribe/Tickets.php

```diff
@@ -1389,5 +1393,15 @@ public static function get_login_url() {
 			 */
 			return apply_filters( 'tribe_tickets_ticket_login_url', $login_url );
 		}
+
+		/**
+		 * @param $return
```

Comment:
It was the pre-existing var name, but I agree that's not very explicit... something like `did_check_in` e `did_uncheck_in`?